docs/docs-freshness-registry.json

```
@@ -2,6 +2,49 @@
+      "last_review": "2026-04-01",
+      "cadence_days": 30
+    },
+    {
+      "path": "tasks/specs/linear-95b040bc-b150-4c39-bc56-64b13f55579f.md",
+      "owner": "Codex (top-level agent), Review agent",
+      "status": "active",
+      "last_review": "2026-04-01",
+      "cadence_days": 30
+    },
+    {
+      "path": "tasks/tasks-linear-95b040bc-b150-4c39-bc56-64b13f55579f.md",
```

Biome (2.4.9)

[error] The key path was already declared.

(lint/suspicious/noDuplicateObjectKeys)